From 71eadcd7e8946dc6e5aea5f9e5826de37b83f60d Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 27 Dec 2022 17:43:26 +0800
Subject: [PATCH] aptiv-s32g: atf-s32g: Patch PFE1-2 SGMII MDIO Bus for GCP

This patch comes from:

recipes-aptiv/u-boot-s32/files/0001-Patch-PFE1-2-SGMII-MDIO-Bus-for-GCP.patch

Upstream-Status: Inappropriate [WR Linux specific change]

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 fdts/s32g.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/fdts/s32g.dtsi b/fdts/s32g.dtsi
index beb306f..4e916cd 100644
--- a/fdts/s32g.dtsi
+++ b/fdts/s32g.dtsi
@@ -1201,13 +1201,13 @@
 
 	pfe1_mdio_pins: pfe1_mdio {
 		pfe1_mdio_grp0 {
-			pinmux = <S32CC_PINMUX(60, FUNC2)>;
+			pinmux = <S32CC_PINMUX(177, FUNC4)>;
 			output-enable;
 			slew-rate = <S32CC_FAST_SLEW_166MHZ>;
 		};
 
 		pfe1_mdio_grp1 {
-			pinmux = <S32CC_PINMUX(61, FUNC2)>;
+			pinmux = <S32CC_PINMUX(178, FUNC4)>;
 			drive-open-drain;
 			output-enable;
 			input-enable;
@@ -1215,7 +1215,7 @@
 		};
 
 		pfe1_mdio_grp2 {
-			pinmux = <S32CC_PINMUX(857, FUNC2)>;
+			pinmux = <S32CC_PINMUX(857, FUNC3)>;
 		};
 
 	};
@@ -1267,7 +1267,7 @@
 
 	pfe2_mdio_pins: pfe2_mdio {
 		pfe2_mdio_grp0 {
-			pinmux = <S32CC_PINMUX(79, FUNC2)>;
+			pinmux = <S32CC_PINMUX(81, FUNC2)>;
 			drive-open-drain;
 			output-enable;
 			input-enable;
@@ -1275,13 +1275,13 @@
 		};
 
 		pfe2_mdio_grp1 {
-			pinmux = <S32CC_PINMUX(82, FUNC2)>;
+			pinmux = <S32CC_PINMUX(80, FUNC1)>;
 			output-enable;
 			slew-rate = <S32CC_FAST_SLEW_166MHZ>;
 		};
 
 		pfe2_mdio_grp2 {
-			pinmux = <S32CC_PINMUX(877, FUNC3)>;
+			pinmux = <S32CC_PINMUX(877, FUNC2)>;
 		};
 
 	};
-- 
2.34.1

